#!/usr/bin/env python
PACKAGE = "imu_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("threshold", double_t, 0, "A double parameter",60, 0,10000)
gen.add("window_size", int_t, 0, "window_size", 20, 2,100)
gen.add("reset",   bool_t,   0, "Reset Variable",  True)
gen.add("overlap",   bool_t,   0, "Overlap Request",  False)

exit(gen.generate(PACKAGE, "imu_ros", "imuLabeler"))
